packages/utils/README.md

```diff
@@ -0,0 +1,127 @@
+# @excalidraw/utils
+
+## Install
+
+    npm i @excalidraw/utils
+
+## API
+
+### `exportToBlob` (async)
+
+Export an Excalidraw diagram to a [Blob](https://developer.mozilla.org/en-US/docs/Web/API/Blob).
+
+### `exportToSvg`
+
+Export an Excalidraw diagram to a [SVGElement](https://developer.mozilla.org/en-US/docs/Web/API/SVGElement).
+
+## Usage
+
+### Webpack
```

Comment:
this can be removed as we need not mention webpack or any bundler and usage can be explained as shown above

Reply:
Since we do not export the library as an ES6 module we do need a bundle. Webpack is probably the most popular and works well with UMD module. Not sure if rollup can also resolve an UMD module as an ES6 module.

Maybe we can be less specific and say "Web bundler"? But we do need to say that the following syntax will only work with a Web bundler:

```js
import { exportToSvg, exportToBlob } from "@excalidraw/utils";
```

Alternatively, we could also publish this package as an ES6 module (see me previous comment).

Reply:
What I meant is it is understood that bundler will be needed when using import syntax so we need not mention it explicitly. If you want to mention `web bundler` that is also fine.

Reply:
> What I meant is it is understood that bundler will be needed when using import syntax so we need not mention it explicitly.

That's not true, you don't need a Web bundler if you are using an ES6 module in a modern browser.
For instance, the following will work out-of-the-box:

```html
<body>
<script type="module">
  import { DateTime } from 'https://cdn.skypack.dev/luxon';
  console.log(DateTime.local());
</script>
</body>
```

I think there's a misconception because Web bundler like Webpack will resolve any type of modules (UMD, ESM, CommonJS...) as if they were ES6 modules.
But that does not mean that you need Webpack to import an ES6 module (at least not anymore).

Reply:
Yup `module` will work out of the box but I am not sure what % completely relies on this as it's not supported in all browsers afaik so I assumed that bundler will be used.